atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -55,9 +52,6 @@ public File acceptorLogDir() {
     @JsonProperty("lockCreator")
     @Value.Default
     public String lockCreator() {
-        log.warn("Lock creator not specified! "
```

Comment:
So this was reverted not long before you commented on it. We do recommend one, but since the default is quite sane, we don't care enough to give a warning here. Once John gets back we can see if he agrees (possibly with another PR if we need to change the warningness).